src/btree/bt_debug.c

```diff
@@ -1269,6 +1272,7 @@ __debug_page_col_var(WT_DBG *ds, WT_REF *ref)
     page = ref->page;
     session = ds->session;
     recno = ref->ref_recno;
+    btree = S2BT(session);
```

Comment:
Nit: When we initialize variables like this, we usually do them in the order that they're declared. So in this case, `btree` should be at the top of this list.

Reply:
Ohh, so here the reason I didn't put btree at the top is because it needs to be below the line where session is initialized

Reply:
Oops. Yeah that makes sense.